ios/PaymentPass.swift

```diff
@@ -1,7 +1,10 @@
 import Foundation
 import PassKit
 
-var pkRequestCallback: RCTResponseSenderBlock?;
+var pkAddPaymentErrorCallback: RCTResponseSenderBlock? = nil
+var pkAddPaymentSuccessCallback: RCTResponseSenderBlock? = nil
+var pkFinaliseSuccessCallback: RCTResponseSenderBlock? = nil
+var pkFinaliseErrorCallback: RCTResponseSenderBlock? = nil
 var pkCompletionHandler: ((PKAddPaymentPassRequest) -> Void)?;
```

Comment:
doesn' this also need to have a default value of nil?

Reply:
done